src/com/phronemophobic/membrane/term.clj

```diff
@@ -313,10 +313,10 @@
                                 :cell-height (tk/font-line-height toolkit term-font)
                                 :descent-gap (- descent-offset baseline-offset)})))))
 
-(defn load-default-toolkit []
+(defn- load-default-toolkit []
   @(requiring-resolve 'membrane.java2d/toolkit))
 
-(def default-color-scheme
+(def ^:private default-color-scheme
```

Comment:
This seems like it would be useful to make public.

Reply:
Yeah, good observation. I shall make it so.

Reply:
I decided to leave this one private and expose run-term and screenshots opts instead.
They both include default-color-scheme.

Reply:
I tend to lean more on the open side for these types of things, especially if they're just data. What's the reason for not making this public?

Reply:
My thinking:
- we are covered because it is included in both `default-run-term-opts` and `default-screensaver-opts`
- it is added to cljdoc docs, and thought it might be superfluous. 

But... can easily expose it if that makes more sense to you. I have no strong feeling on the matter. If it will be useful to you or others, let's expose it.

Reply:
@phronmophobic lemme know your choice on this one, and I shall make it so.

Reply:
To me, it makes sense to expose it!

Reply:
Very good sir! It is done.